app/views/projects/index.html.erb

```diff
@@ -6,6 +6,19 @@
         <%= project.owner_name %>
         <%= project.repo_url %>
         <%= project.live_preview %>
+        <%= project.total_number_of_votes %>
+
+        <%= button_to lesson_project_votes_path(project.lesson.id, project.id),
```

Comment:
`project.lesson` will fire a query right? Can't we just use `project.lesson_id`?